source/NVDAObjects/window/excel.py

```diff
@@ -72,6 +72,99 @@
 xlCellTypeSameValidation      =-4175      # from enum XlCellType
 xlCellTypeVisible             =12         # from enum XlCellType
 
+#Excel Cell Patterns
+xlPatternAutomatic           =-4105       # from enum XlPattern
+xlPatternChecker             =9           # from enum XlPattern
+xlPatternCrissCross          =16          # from enum XlPattern
+xlPatternDown                =-4121       # from enum XlPattern
+xlPatternGray16              =17          # from enum XlPattern
+xlPatternGray25              =-4124       # from enum XlPattern
+xlPatternGray50              =-4125       # from enum XlPattern
+xlPatternGray75              =-4126       # from enum XlPattern
+xlPatternGray8               =18          # from enum XlPattern
+xlPatternGrid                =15          # from enum XlPattern
+xlPatternHorizontal          =-4128       # from enum XlPattern
+xlPatternLightDown           =13          # from enum XlPattern
+xlPatternLightHorizontal     =11          # from enum XlPattern
+xlPatternLightUp             =14          # from enum XlPattern
+xlPatternLightVertical       =12          # from enum XlPattern
+xlPatternNone                =-4142       # from enum XlPattern
+xlPatternSemiGray75          =10          # from enum XlPattern
+xlPatternSolid               =1           # from enum XlPattern
+xlPatternUp                  =-4162       # from enum XlPattern
+xlPatternVertical            =-4166       # from enum XlPattern

```

Comment:
nit: These lines all have this comment at the end:

`# from enum XlPattern`

That could just be part of the comment at the top of all of them (if it's even needed at all), rather than being on every line.